hash/msx2_flop.xml

```diff
@@ -9917,6 +12026,8 @@ The following floppies came with the machines.
 		<description>Rune Master III (Japan, alt)</description>
 		<year>1991</year>
 		<publisher>Compile</publisher>
+		<info name="alt_title" value="ルーンマスター三国英傑伝"/>
```

Comment:
Why is the subtitle in the description of the parent but not the clones for this game?

Reply:
I hadn't noticed that, thanks. Fixed.